.azure-pipelines.yml

```diff
@@ -30,32 +30,8 @@ variables:
 - template: build/variables.yml
 
 stages:
-#-if false
```

Comment:
Why would you remove this?

Reply:
This section tests that the dotnet new command works and generates an app that actually builds.